data/blog.yml

```diff
@@ -3,4 +3,4 @@ sharing : true
 staticmanComments:
   enable: false
   notifications: true
-  api: https://api.staticman.net/v3/entry/github/thecloudnatives/website/master/comments
```

Comment:
@jocatalin I assume this will need to be updated.

Reply:
We didn't enable staticman, at the time we looked at it there was some discussion about application being banned from Github because of spam.
I added email notification on Netlify, the following should enable it:
```<form name="contact" id="contact-form" class="form-meghna" method="post" data-netlify="true" role="form">```

Reply:
I had a look & it doesn't look that easy to get this setup. I was trying to add the bot as a contributor, but it looks like it may still be banned.